dashboard/app/models/ability.rb

```diff
@@ -132,6 +133,7 @@ def initialize(user)
         can :manage, :pd_workshop_summary_report
         can :manage, :pd_teacher_attendance_report
         can :manage, Pd::TeacherApplication
+        can :manage, :pd_workshop_user_admin
```

Comment:
How about `workshop_user_management` ?

It's not a big deal, but I tend to read `"admin"` as administrator, like our `User.admin?` property rather than "administer".